src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderValue.cs

```diff
@@ -22,12 +29,23 @@ private static CoseHeaderValue FromEncodedValue(ReadOnlyMemory<byte> encodedValu
             return value;
         }
 
+        /// <summary>
+        /// Creates a <see cref="CoseHeaderValue"/> instance from an encoded CBOR value provided as a <see cref="ReadOnlySpan{T}"/>.
+        /// </summary>
+        /// <param name="encodedValue">The encoded value that the instance will represent.</param>
```

Comment:
This feels a little clunky, but I'm having trouble prettying it up.

e.g.

```suggestion
        /// <param name="encodedValue">The bytes of a CBOR-encoded value to... </param>
```

isn't great, because it's about to assume we're adding it to a CoseHeaderMap.

Maybe just

```suggestion
        /// <param name="encodedValue">A CBOR-encoded value.</param>
```

Reply:
Second one looks better to me since the summary already explains what encodedValue is for.

Reply:
Btw, would you rather have "CBOR-encoded" or "CBOR encoded"? The no dash version is already used in one S.F.Cbor API, but it can be changed.

Reply:
https://www.grammarly.com/blog/hyphen/

Since CBOR modifies encoded and they precede the noun, I believe `CBOR-encoded` is correct.  If we have anywhere that it follows the noun ("the value was not CBOR encoded correctly") it doesn't get a hyphen then... apparently :smile:.